docs/sources/dashboards/create-reports/report-settings/index.md

```diff
@@ -11,46 +11,46 @@ menuTitle: Settings
 title: Reporting settings
 description: Manage organizational Reporting settings
 weight: 700
-refs:
-  grafana-enterprise:
-    - pattern: /docs/grafana/
-      destination: /docs/grafana/<GRAFANA_VERSION>/introduction/grafana-enterprise/
-    - pattern: /docs/grafana-cloud/
-      destination: /docs/grafana/<GRAFANA_VERSION>/introduction/grafana-enterprise/
-  configuration:
-    - pattern: /docs/grafana/
-      destination: /docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana/#filters
-    - pattern: /docs/grafana-cloud/
-      destination: /docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana/#filters
 ---
 
 # Reporting settings
 
-You can configure organization-wide report settings and branding options in **Dashboards > Reporting > Settings**. Settings are applied to all the reports for the current organization.
+You can configure organization-wide report settings and branding options in **Dashboards > Reporting > Settings**.
+Settings are applied to all the reports for the current organization.
 
 ## Attachment settings
 
 The options in this section control the branding and theming of the report attachments.
 
 ### PDF
 
-- **Company logo** - Company logo displayed in the report PDF. It can be configured by specifying a URL, or by uploading a file. The maximum file size is 16 MB. Defaults to the Grafana logo.
+- **Company logo** - Company logo displayed in the report PDF.
+  Configure it by specifying a URL or uploading a file.
+  The maximum file size is 16 MB.
+  Defaults to the Grafana logo.
 
-- **Theme** - Theme of the PDF attached to the report. Defaults to **Light**. The selected theme is also applied to the PDFs generated when you click **Preview PDF** during report creation or select the **Export as PDF** option on a dashboard. If **Current** is selected, the PDF in the report will be in the instance theme of the Admin, but the preview and exported PDFs will be in the user's instance theme.
+- **Theme** - Theme of the PDF attached to the report.
+  The selected theme is also applied to the PDFs generated when you click **Preview PDF** during report creation or select the **Export as PDF** option on a dashboard.
+
+  If **Current** is selected, the PDF in the report is in the instance theme of the organization Admin, but the preview and exported PDFs are in the user's instance theme.
+  Defaults to **Light**.
 
 ### Embedded Image
 
-- **Theme** - Theme of the dashboard image embedded in the email. Defaults to **Dark**.
+- **Theme** - Theme of the dashboard image embedded in the email.
+  If **Current** is selected, the image in the report is in the instance theme of the organization Admin.
```

Comment:
```suggestion
  If **Current** is selected, the image in the report is in the instance theme of the report creator.
```
If the report creator has no theme set, then, it will fall back on the team or organization or server theme but I don't know if we should go into this much detail. We may want to point toward this doc as well: https://grafana.com/docs/grafana/latest/administration/organization-preferences/#change-grafana-ui-theme